src/reducers/duplicatesReducer.js

```diff
@@ -0,0 +1,41 @@
+import initialState from "./initialState";
+import {
+    FETCH_DUPLICATES_SUCCESS,
+    IGNORE_DUPLICATES,
+    IGNORE_DUPLICATES_SUCCESS,
+    MERGE_DUPLICATES,
+    MERGE_DUPLICATES_SUCCESS
+} from "../constants/actions";
+
+const duplicatesReducer = (state = initialState.duplicates, action) => {
+    switch (action.type) {
+        case FETCH_DUPLICATES_SUCCESS:
+            return action.payload.duplicates;
+        case MERGE_DUPLICATES:
+            return state.map((duplicate) => {
```

Comment:
I encourage you to use lodash whenever you can. It has better performance than native JavaScript functions.